### DISCUSSION TITLE
github error suspension！

### DISCUSSION THREAD

**Discussion by byqx999** [community/community#140836] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, according to the information I received, using copilot in vscode to link to a github account is extremely likely to cause system misjudgment and cause an account to be suspended by mistake. I was suspended for no apparent reason without successfully connecting and using copilot, which confirms the previous speculation of GitHub users. This is a very serious problem! My github account has very few repositories, and they are all codes used by my scientific research institute. I have never published any illegal content, and I am also using it very formally and carefully. This should not be blocked by the system due to the copilot plug-in; however, when I posted a ticket for negotiation, I did not get any feedback from the github community managers for a long time, which made me feel quite disappointed. You know, there are many similar wrong suspensions, and the scope of influence is very large. My ticket id is 3034214. I strongly hope that the misjudgment represented by this can attract your attention and timely processing. Thank you very much for your cooperation.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @byqx999, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.!_
